.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please clean the IDE generated configuration files. This would help us to do better code review.